components/timezone/src/metazone.rs

```diff
@@ -19,38 +19,47 @@ pub struct MetazoneCalculator {
     pub(super) metazone_period: DataPayload<MetazonePeriodV1Marker>,
 }
 
+#[cfg(feature = "data")]
+impl Default for MetazoneCalculator {
```

Comment:
thought: Not quite sure we should do this.

Reply:
https://rust-lang.github.io/rust-clippy/v0.0.212/index.html#new_without_default

Reply:
aha, I see. hm. Makes sense I guess.